src/mbgl/tile/raster_dem_tile.hpp

```diff
@@ -0,0 +1,57 @@
+#pragma once
+
+#include <mbgl/tile/tile.hpp>
+#include <mbgl/tile/tile_loader.hpp>
+#include <mbgl/tile/raster_dem_tile_worker.hpp>
+#include <mbgl/actor/actor.hpp>
+
+namespace mbgl {
+
+class Tileset;
+class TileParameters;
+class HillshadeBucket;
+
+namespace style {
+class Layer;
+} // namespace style
+
+class RasterDEMTile : public Tile {
+public:
+    RasterDEMTile(const OverscaledTileID&,
+                   const TileParameters&,
+                   const Tileset&);
+    ~RasterDEMTile();
+
+    void setNecessity(TileNecessity) final;
+
+    void setError(std::exception_ptr);
+    void setMetadata(optional<Timestamp> modified, optional<Timestamp> expires);
+    void setData(std::shared_ptr<const std::string> data);
+
+    void cancel() override;
```

Comment:
Give `Tile::cancel` an empty implementation so this override and the on in `RasterTile` are not needed.